clipper_admin_v2/clipper_admin/container_manager.py

```diff
@@ -0,0 +1,87 @@
+import abc
+
+# Constants
+
+CLIPPER_QUERY_PORT = 1337
```

Comment:
These must not be hardcoded for k8s compatibility, where there is no guarantee of the actual port bound by a service. Since we don't ever use these without having a reference to a CM, consider making this a property of a running CM instead